File: app/api/stats/stats.js
'use strict';

// Aggregate server statistics: version, uptime, total users, rooms and active calls
const url = 'http://localhost:8000/api/v1/stats';

const authorization = 'call_me_api_key_secret';

fetch(url, {
    method: 'GET',
    headers: {
        Authorization: authorization,
        'Content-Type': 'application/json',
    },
})
    .then((response) => response.json())
    .then((data) => console.log(data))
    .catch((error) => console.error('Error:', error));

File: app/api/stats/stats.py
import requests # pip3 install requests

# Aggregate server statistics: version, uptime, total users, rooms and active calls
url = "http://localhost:8000/api/v1/stats"

authorization = "call_me_api_key_secret"

headers = {
    'Authorization': authorization,
    'Content-Type': 'application/json'
}

response = requests.get(url, headers=headers)

print(response.json())
